cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml

```diff
@@ -117,6 +117,14 @@
       <artifactId>twitter4j-stream</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-mapreduce-client-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>mysql</groupId>
+      <artifactId>mysql-connector-java</artifactId>
```

Comment:
we'll still need to package it though right? Since per my understanding the built in sources and sinks won't accept jar files? I'll add a note, and we can fix it when we work on packaging.